.hound.yml

```diff
@@ -0,0 +1,2 @@
+jshint:
```

Comment:
Are you using jslint or eslint for this project? Please ask question when you need to. Please correct this before it can be merged